.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
   global:
     - NODE_ENV=test
 before_script:
+  - bundle exec rake db:create RAILS_ENV=test
   - bundle exec rake db:schema:load RAILS_ENV=test
```

Comment:
I was getting an error with TravisCI, saying `grumbler_test` didn't exist, so I created it before, and it has fixed the issue.